qiskit/pulse/library/parametric_pulses.py

```diff
@@ -341,23 +341,24 @@ class Drag(ParametricPulse):
     """The Derivative Removal by Adiabatic Gate (DRAG) pulse is a standard Gaussian pulse
     with an additional Gaussian derivative component and lifting applied.
 
-    It is designed to reduce the frequency spectrum of a normal gaussian pulse near
+    It is designed to reduce the frequency spectrum of a standard Gaussian pulse near
     the :math:`|1\\rangle\\leftrightarrow|2\\rangle` transition,
     reducing the chance of leakage to the :math:`|2\\rangle` state.
 
     .. math::
 
         g(x) &= \\exp\\Bigl(-\\frac12 \\frac{(x - \\text{duration}/2)^2}{\\text{sigma}^2}\\Bigr)\\\\
-        f'(x) &= g(x) + 1j \\times \\text{beta} \\times \\frac{\\mathrm d}{\\mathrm{d}x} g(x)\\\\
-              &= g(x) + 1j \\times \\text{beta} \\times\
-                    \\Bigl(-\\frac{x - \\text{duration}/2}{\\text{sigma}^2}\\Bigr)g(x)\\\\
-        f(x) &= \\text{amp}\\times\\frac{f'(x)-f'(-1)}{1-f'(-1)}, \\quad 0 \\le x < \\text{duration}
+        g'(x) &= \\text{amp}\\times\\frac{g(x)-g(-1)}{1-g(-1)}\\\\
+        f(x) &=  g'(x) \\times \\Bigl(1 + 1j \\times \\text{beta} \\times\
+            \\Bigl(-\\frac{x - \\text{duration}/2}{\\text{sigma}^2}\\Bigr)  \\Bigr),
+            \\quad 0 \\le x < \\text{duration}
 
-    where :math:`g(x)` is a standard unlifted gaussian waveform and
-    :math:`f'(x)` is the DRAG waveform without lifting or amplitude scaling.
+    where :math:`g(x)` is a standard unlifted Gaussian waveform and
+    :math:`g'(x)` is the lifted 
+    `Gaussian <https://qiskit.org/documentation/stubs/qiskit.pulse.library.Gaussian.html>`_ waveform.
```

Comment:
```suggestion
    :class:`~qiskit.pulse.library.Gaussian` waveform.
```
since this page lives in the same apidocs, you can directly jump to the class api docs like this.